### DISCUSSION TITLE
Adjuest alarm record message length.

### DISCUSSION THREAD

**Discussion by weixiang1862** [apache/skywalking#11402] [General]
AlarmRecord's alarmMessage column use default length 200, which is very easy to reach the upper limit when we use full agent name `group:service|namespace|cluster`, Should we adjuest it ?

![image](https://github.com/apache/skywalking/assets/16346644/17f66287-2957-46cd-a130-7fc5056ccf46)
![image](https://github.com/apache/skywalking/assets/16346644/02f85ffa-861f-4397-93ba-0493362b4683)

**Comment by wu-sheng** [comment]
How long would be your message?

**Reply by weixiang1862** [reply]
It is an endpoint relation alarm_message which has 196 characters (sorry for pic, it's our customer remote virtual desktop, i can not copy text from it).
![image](https://github.com/apache/skywalking/assets/16346644/b58a5e99-85bf-4967-ba63-230730d271b4)

**Reply by wu-sheng** [reply]
How long would you expect to be enough?

**Reply by weixiang1862** [reply]
256 is enough in our customer system, they don‘t hava longer agent name or endpoint name because it almostly lost readability if for longer.

Considering alarm record is important and it won‘t grow into a large amount of data in normal system, I think 512 is better if we take some extreme cases into account.

**Reply by wu-sheng** [reply]
I think 200->512 is acceptable.

**Reply by weixiang1862** [reply]
Ok, thanks. I will submit it after I do some tests.
